test/test_jit.py

```diff
@@ -15290,7 +15290,7 @@ def foo(self, x1, x2):
                 return torch.neg(x1), self.param, self.const, torch.neg(x2), self.param
 
             @torch.jit.script_method
-            def wait_script(self, x1, x2):
+            def forward(self, x1, x2):
```

Comment:
why did this change?

Reply:
EmitModuleHook now compares the `.code` of the modules instead of relying on internal details of the namespace. But `.code` gives you the `forward()` method, otherwise it doesn't work.